Sources/UnidirectionalBinding.swift

```diff
@@ -276,3 +277,62 @@ extension BindingTarget where Value: OptionalProtocol {
 		return target <~ property.producer
 	}
 }
+
+/// A pseudo `BindingTarget`, constructed from a closure and a lifetime.
```

Comment:
What do you think about dropping the word "pseudo"? To me it makes it sound like this isn't a "real" binding target, even though it's as real as any other binding target (and arguably purer).